source/MRMesh/MRFlagOperators.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <type_traits>
```

Comment:
?

Reply:
The macros use `std::underlying_type`. I got an error about it not being defined, hence adding the header.